connect/runtime/src/main/java/org/apache/kafka/connect/storage/KafkaConfigBackingStore.java

```diff
@@ -291,7 +328,16 @@ public void start() {
         log.info("Starting KafkaConfigBackingStore");
         // Before startup, callbacks are *not* invoked. You can grab a snapshot after starting -- just take care that
         // updates can continue to occur in the background
-        configLog.start();
+        try {
+            configLog.start();
+        } catch (UnsupportedVersionException e) {
+            throw new ConnectException(
+                    "Enabling exactly-once support for source connectors requires a Kafka broker version that allows "
+                            + "admin clients to read consumer offsets. Disable the worker's exactly-once support "
+                            + "for source connectors, or use a new Kafka broker version.",
```

Comment:
Haha yep, caught and fixed this in an upstream PR that's since been merged. Will pick up in the rebase.

https://github.com/apache/kafka/blob/a6c5a74fdbdce9a992b47706913c920902cda28c/connect/runtime/src/main/java/org/apache/kafka/connect/storage/KafkaConfigBackingStore.java#L323

Reply:
Rebase complete; should be resolved now.